autoload/neocomplete/handler.vim

```diff
@@ -171,17 +171,21 @@ function! neocomplete#handler#_on_text_changed() "{{{
     call s:make_cache_current_line()
   endif
 
-  let cur_text = matchstr(getline('.'), '^.*\%'.col('.').'c')
   " indent line matched by indentkeys
-  for word in filter(map(split(&l:indentkeys, ','),
+  let neocomplete = neocomplete#get_current_neocomplete()
+  let cur_text = matchstr(getline('.'), '^.*\%'.col('.').'c')
+  if get(neocomplete, 'indent_text', '') != matchstr(getline('.'), '\S.*$')
```

Comment:
I think indent_text should be initialized in `neocomplete#init#_current_neocomplete()`.